src/Form/matching.js

```diff
@@ -410,7 +410,7 @@ class Matching {
             return { matched: false }
         }
         /** @type {MatchableStrings[]} */
-        const stringsToMatch = ['nameAttr', 'labelText', 'placeholderAttr', 'id', 'relatedText']
+        const stringsToMatch = ['placeholderAttr', 'nameAttr', 'labelText', 'id', 'relatedText']
```

Comment:
I've switched to check the placeholder first, as that seem to yield positive effects overall. The idea is that what the user sees is more accurate than what developers put in the code (which is unfortunate, but 🤷).